jquery.edittable.js

```diff
@@ -133,7 +133,13 @@
 
             checkButtons();
         }
-
+        
+        // Update settings - header , template
+        function updateSettings(header = null, template = null) {
```

Comment:
Rename `header`/`template `to `headerCols` and `rowTemplate`. Also in documentation.

Reply:
And on line 313

Reply:
Done